### PR TITLE
Camera feed fallback protocol on IOS devices

### LINKED ISSUE #2917: Build Fallback Protocol to Support iOS Devices
The Streaming Server currently Supports HLS along with MSE. We need to use HLS as a fallback on devices such as iOS where MSE is not supported

### PR DESCRIPTION
**Pull request by rabilrbl**
Resolve #2917 

**Changes**
- Extra HLS player for IOS devices as MSE is not supported

**Not working**
- Fullscreen toggle for HLS player ( will be taken as seperate issue )

### CODE REVIEW

**Comment by gigincg** [comment]
@rabilrbl Isn't this ready to be reviewed/merged

**Comment by rabilrbl** [comment]
@gigincg Stuck with fullscreen not toggling.

**Comment by mathew-alex** [comment]
:wave: Hi, @rabilrbl,
Conflicts have been detected against the base branch. Please rebase your branch against the base branch.

***

This message is automatically generated by [prince-chrismc/label-merge-conflicts-action](https://github.com/prince-chrismc/label-merge-conflicts-action) so don't hesitate to report issues/improvements there.

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/Consultations/Feed.tsx [hunk 1]
```diff
@@ -294,11 +307,13 @@ export const Feed: React.FC<IFeedProps> = ({ consultationId }) => {
...
-              ? videoWrapper.current
-              : liveFeedPlayerRef.current
-          )
+        ? isIOS
+          ? screenfull.request(
+              videoWrapper.current
+                ? videoWrapper.current
+                : (liveFeedPlayerRef.current as HTMLElement)
+            )
+          : screenfull.request(findDOMNode(liveFeedPlayerRef.current) as any)
```
Comment (khavinshankar) [inline src/Components/Facility/Consultations/Feed.tsx:316]: why are we doing it differently for iOS and other devices, is this condition necessary?

and also shouldn't it be the other way around?
```suggestion
        ? !isIOS
          ? screenfull.request(
              videoWrapper.current
                ? videoWrapper.current
                : (liveFeedPlayerRef.current as HTMLElement)
            )
          : screenfull.request(findDOMNode(liveFeedPlayerRef.current) as any)
```

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
merging this pr for now, if there is any follow-up that involves code change, feel free to create a new pr

**Pull request merged**